packages/graph-engine/src/nodes/color/lib/sortColors.ts

```diff
@@ -27,8 +27,12 @@ export const sortTokens = (colors, compareColor, type, algorithm) =>
 
 			return {
 				color,
-				compareValue
+				compareValue,
+				index
 			};
 		}),
 		['compareValue']
-	).map(color => color.color);
+	).map(item => ({
```

Comment:
You might as well just remove this in this case because you're now iterating over the array 3 times